src/PlusDataCollection/Andor/vtkPlusAndorVideoSource.cxx

```diff
@@ -928,6 +935,12 @@ PlusStatus vtkPlusAndorVideoSource::SetBiasDarkCorrectionImage(const std::string
     {
       throw "Bias+dark correction image empty!";
     }
+    if (cvBiasDarkCorrection.cols != frameSize[0] || cvBiasDarkCorrection.rows != frameSize[1])
+    {
+      LOG_ERROR("BiasDarkCorrectionImage size " << cvBiasDarkCorrection.size()
+        << " does not match sensor size " << frameSize[0] << " x " << frameSize[1]);
+      return PLUS_FAIL;
+    }
```

Comment:
If the sensor size is 2048x2048 and I start the server with no binning, and then I decided to take a capture with binning=2, I'm unable to call `SetBiasDarkCorrectionImage` to specify the 1024x1024 calibration image prior to calling `AcquireBLI` with binning of 2. I think you should fail during `ApplyFrameCorrections` which happens post `AcquireBLI` to avoid the sudden crash of using a mismatched correction image frame size.

This goes for all these correction image error checks added.

Reply:
I put error logging into `ApplyFrameCorrections` and changed these into `INFO`.